index.html

```diff
@@ -40,6 +40,222 @@ <h1>I'm Alishah. Glad to see you!</h1>
+                            <span class="canopy">CANOPY</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">Back End Dev</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">2015</span>
+                        </div>
+                    </div>
+                    <div class="primarytext2">
+                        <span>A daily selection of privately</span>
+                        <span>personalized reads;no accounts or</span>
+                        <span> sign-ups required.</span>
+                    </div>
+                    <div class="primarytext3">
+                        <button class="lang">HTML</button>
+                        <button class="lang">CSS</button>
+                        <button class="lang JavaScript">JavaScript</button>
+                    </div>
+                    <div class="primarytext4">
+                        <button class="project">See project</button>
+                    </div>
+                </div>
+            </div>
+            <div class="works-container">
+                <div class="snapshoot">
+                    <img src="./Image/SnapshootPortfolio(5).png" alt="">
+                </div>
+                <div class="leftBlock">
+                    <div class="primarytext1">
+                        <div class="multi-post">
+    
+                            Multi-Post
+                            <br>
+                            Stories
+    
+                        </div>
+    
+                        <div class="fram">
+                            <span class="canopy">CANOPY</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">Back End Dev</span>
+                            <img src="./Image/Counter.png" alt="center">
+                            <span class="num-text">2015</span>
+                        </div>
+                    </div>
+                    <div class="primarytext2">
+                        <span>A daily selection of privately</span>
+                        <span>personalized reads;no accounts or</span>
+                        <span> sign-ups required.</span>
+                    </div>
+                    <div class="primarytext3">
+                        <button class="lang">HTML</button>
+                        <button class="lang">CSS</button>
+                        <button class="lang JavaScript">JavaScript</button>
+                    </div>
+                    <div class="primarytext4">
+                        <button class="project">See project</button>
+                    </div>
+                </div>
+            </div>
+            <div class="about-my-slef">
+                <div class="top">
+                    <div class="first-text">About <br> Myself</div>
```

Comment:
- [ ]  I will recommend you ensure that the about me title is in heading tags e.g (`<h2>`) and not a `div` tag because it represents the title of the section and as such the heading semantic tag should be used so that screen readers and browsers will be able to understand what every part of your website represents 👍🏽